utils/src/main/scala/com/salesforce/op/utils/io/avro/AvroInOut.scala

```diff
@@ -154,6 +155,8 @@ object AvroInOut {
 
   implicit class AvroWriter[T <: GenericRecord](rdd: RDD[T]) {
 
+    private implicit val job = Job.getInstance(rdd.sparkContext.hadoopConfiguration)
```

Comment:
On line `167` you can also remove `implicit job: Job` in `def writeAvro(path: String)`